devel/icu/Portfile

```diff
@@ -64,8 +64,13 @@ set docdir      ${prefix}/share/doc/${name}
 
 configure.args              --disable-layoutex \
                             --disable-samples \
+                            --disable-icu-config \
```

Comment:
This changes the files installed by the port, so it would be necessary to increase the revision of all of the subports that are affected by this change.